src/eventDispatchers/mouseEventHandlers/mouseMove.js

```diff
@@ -22,10 +22,28 @@ export default function (evt) {
   let tools;
   const { element, currentPoints } = evt.detail;
 
+  // Set the mouse position incase any tool needs it.
+  setters.mousePositionImage = currentPoints.image;
```

Comment:
I was winging the implementation for `getters`, `state`, `mutations`, but we can switch to `setters` if that makes more sense? Honestly, I was stealing some nomenclature from Vue/React

Reply:
I'm easy on this one, I just added some setting function as I wasn't so sure this counted as a mutation (perhaps it does?), and needed somewhere to put it.

Reply:
Sounds good. We can merge this as is, and I can update it to match the current method?

Reply:
If you check the store, I've already done this :).